src/pages/resultsView/plots/PlotsTab.tsx

```diff
@@ -4809,7 +4809,11 @@ export default class PlotsTab extends React.Component<IPlotsTabProps, {}> {
                                 <Observer>{this.controls}</Observer>
                             ) : (
                                 <LoadingIndicator
-                                    isLoading={true}
+                                    isLoading={
```

Comment:
we should probably package this up with other loading states (e.g. dataTypeOptions.pending) 
 i.e. it's odd that we show don't show loader for those promises, but do for this one.